lib/twitch/client.rb

```diff
@@ -38,96 +39,84 @@ class Client
 
     private_constant :TOKENS_CONFLICT_WARNING
 
+    attr_reader :tokens
+
     # Initializes a Twitch client.
     #
     # - client_id [String] The client ID.
     # Used as the Client-ID header in a request.
+    # - client_secret [String] The client secret.
+    # Used for generation access tokens.
+    # - redirect_uri [String] A redirect URI.
+    # Used for redirection after successful authentication.
+    # - scopes [Array<String>] Required scopes.
+    # Used for possibility to make planned requests.
```

Comment:
Would rather say something along the lines of 
>Used to specify permissions for the token.

Reply:
OK, done.